lib/Firtool/Firtool.cpp

```diff
@@ -133,6 +134,7 @@ LogicalResult firtool::populateCHIRRTLToLowFIRRTL(mlir::PassManager &pm,
 
   // Must run this pass after all diagnostic passes have run, otherwise it can
   // hide errors.
```

Comment:
Mega-nit: can you update the comment to indicate that this is plural?

Reply:
Updated the comment.